packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosure.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import {useDisclosurePrimitiveState} from '@twilio-paste/disclosure-primitive';
+import type {DisclosurePrimitiveInitialState} from '@twilio-paste/disclosure-primitive';
+
+import type {SidebarCategoryRoutes} from '../../../../constants';
+import {getCurrentPathname} from '../../../../utils/RouteUtils';
+
+import {SidebarItem} from '../SidebarItem';
+import {SidebarDisclosureContext} from './SidebarDisclosureContext';
+
+interface SidebarDisclosureProps extends DisclosurePrimitiveInitialState {
+  children: React.ReactNode;
```

Comment:
NIT: If the children cannot be falsey, we could consider using `NonNullable<React.ReactNode>` to exclude `null` and `undefined` from the type definition of `React.ReactNode`